### PR TITLE
D8UN-623 Add current page to all breadcrumb links

### PR DESCRIPTION
**Pull request by ooneill547**
This needed to be completed for all unity sites

### CODE REVIEW

File: web/sites/fiscalcommissionni/modules/custom/fiscalcommissionni_breadcrumbs/src/PageBreadcrumb.php [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+    return $match;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function build(RouteMatchInterface $route_match) {
+    $breadcrumb = new Breadcrumb();
+    $title_resolver = $this->titleResolver->getTitle($this->request->getCurrentRequest(), $route_match->getRouteObject());
+    if ($this->node) {
```
Comment (omahm) [inline web/sites/fiscalcommissionni/modules/custom/fiscalcommissionni_breadcrumbs/src/PageBreadcrumb.php:105]: Shouldn't really need this check as build will only get called if we have passed the applies() call

**Review by omahm — CHANGES_REQUESTED** [review]
Generally looks good, just a couple of issues.

**Comment by ooneill547** [comment]
> Generally looks good, just a couple of issues.

I have updated the fiscal commission breadcrumbs to check if it's correct and then I will do the rest

**Comment by johangant** [comment]
I think this could do with an update/merge from `development` to assess what has changed; there are a number of things that look like they come from other feature branches and should already be in `development` by now.

**Review by omahm — APPROVED** [review]
Looks good

**Pull request merged**